packages/d3fc-webgl/src/buffers/elementConstantAttributeBuilder.js

```diff
@@ -10,9 +11,9 @@ export default () => {
     const project = (elementCount, verticesPerElement) => {
         const components = base.size();
         const offset = base.offset();
-        const projectedData = new Float32Array(
-            offset + elementCount * verticesPerElement * components
-        );
+        const requiredLength =
+            offset + elementCount * verticesPerElement * components;
+        const projectedData = floatArrayFactory(data, requiredLength);
```

Comment:
`data` is not a `Float32Array` - it's just a standard javascript list. I think that @cprice meant that we should recycle `projectedData`.

Reply:
As discussed, `data` is actually a `Float32Array`, but you might be right about re-using projected data between attributeBuilders, if that's possible. Let's get more info and I can make changes if need be.

Reply:
Changed the whole thing, I'll resolve this for now.